openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -21,22 +22,18 @@
 from openpype.pipeline.create import get_legacy_creator_by_name
 
 
-def load_subset(
-    project_name, asset_name, subset_name, loader_type=None, ext="blend"
-):
-    """Load the representation of the last version of subset.
+def download_subset(project_name, asset_name, subset_name, ext="blend"):
+    """Download the representation of the last version of subset on current site.
```

Comment:
```suggestion
    """Download the representation of the subset last version on current site.
```
I think this can be worded in a simpler way than 'of of'. Also I wonder if 'local site' would be more explicit than 'current site'?
For this one (not the others like that I found) It's also two characters too long, but if you apply my suggestion it's going to be short enough.